account/src/main/java/com/user/account/advice/CustomExceptions.java

```diff
@@ -0,0 +1,9 @@
+package com.user.account.advice;
+
+
+public class CustomExceptions extends Exception{
```

Comment:
Very general name, try and give a specific name like `UserNotFoundException`

Reply:
Change as recommended